### PR TITLE
test: Reset DB and skip set-up before running node creator e2e

### PR DESCRIPTION
**Pull request by OlegIvaniv**

### CODE REVIEW

File: cypress/e2e/4-node-creator.cy.ts [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-const username = DEFAULT_USER_EMAIL;
-const password = DEFAULT_USER_PASSWORD;
-const firstName = randFirstName();
-const lastName = randLastName();
 const nodeCreatorFeature = new NodeCreator();
 
 describe('Node Creator', () => {
+	before(() => {
+		cy.task('db:reset');
+	})
```
Comment (ivov) [inline cypress/e2e/4-node-creator.cy.ts:10]: Why is this needed? None of the others reset in `before`, plus `beforeEach` should cover it?

File: cypress/e2e/4-node-creator.cy.ts [hunk 2]
```diff
@@ -24,16 +22,7 @@ describe('Node Creator', () => {
...
-		cy.signup(username, firstName, lastName, password);
-
-		cy.on('uncaught:exception', (err, runnable) => {
-			expect(err.message).to.include('Not logged in');
-
-			return false;
-		})
-
-		cy.signin(username, password);
+		cy.task('db:reset');
```
Comment (ivov) [inline cypress/e2e/4-node-creator.cy.ts:25]: As a precaution, resetting the DB should be the first instruction in `beforeEach`.

**Review by ivov — COMMENTED** [review]
For reference, why did you decide to skip signup?

File: cypress/e2e/4-node-creator.cy.ts (hunk 1 above)
Comment (OlegIvaniv) [reply cypress/e2e/4-node-creator.cy.ts:10]: The other `cy.task` was unnecessary, forgot to delete it.

File: cypress/e2e/4-node-creator.cy.ts (hunk 2 above)
Comment (OlegIvaniv) [reply cypress/e2e/4-node-creator.cy.ts]: In the specs, there's no saving of the workflow, or user management etc. So its seems safe to just do it once before running all the tests; fewer operations to do > faster spec.

**Comment by OlegIvaniv** [comment]
> For reference, why did you decide to skip signup?

Because the focus of this test is the node panel, the login is tested in other specs. And the setup/non-setup doesn't matter for the node creator.

**Review by ivov — APPROVED** [review]
All passing locally.

**Comment by janober** [comment]
Got released with `n8n@0.205.0`
